test/integration/targets/win_psexec/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+- name: Download PSTools
+  win_get_url:
+    url: https://download.sysinternals.com/files/PSTools.zip
+    dest: C:\Windows\Temp\PSTools.zip
+    force: no
+
+- name: Create sysinternals directory
+  win_file:
+    path: C:\Program Files\sysinternals
```

Comment:
Good point, I just always do it that way :-) Doing it in tempdir also would require me to create a directory, as the zip files is a blob of binaries with no directory. Will fix !

Actually I don't understand why Microsoft doesn't properly package it...